students/kondel_tomasz/lesson_01_basics/Clock_face_1.py

```diff
@@ -0,0 +1,21 @@
+import math
+H = int(input())
+M = int(input())
+S = int(input())
+
+total = 60 * 60 * 12
+current = S + (M * 60) + (H * 60 * 60)
+ang = (current * 360) / total
+x = round(ang, 5) - ang
+per = (x * 10000000) / ang
+if per == 0:
+    if int(ang) == ang:
+        print(int(ang))
+    else:
+        print(ang)
+elif math.fabs(per) < 1:
+    print(round(ang, 3))
+elif math.fabs(per) < 100:
+    print(round(ang, 5))
+else:
+    print(round(ang, 8))
```

Comment:
The thing you are looking for is `print('{:g}'.format(ang))`. I don't think they expected precision to be handled in that task. Quite a feat to make it pass, anyway :+1: